src/index.html

```diff
@@ -5,8 +5,391 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="header">
+        <a href="#home" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+  
+        <nav class="navigation">
+          <ul>
+            <li class="navigation__li">
```

Comment:
these are names, and the word "li" by itself does not make any sense as a class name
```suggestion
            <li class="navigation__item">
```